libclamav/others.c

```diff
@@ -204,8 +211,11 @@ static void *load_module(const char *name, const char *featurename)
 
     for (i = 0; i < sizeof(suffixes) / sizeof(suffixes[0]); i++) {
         snprintf(modulename, sizeof(modulename), "%s" PATHSEP "%s%s", SEARCH_LIBDIR, name, suffixes[i]);
-
         rhandle = dlopen(modulename, RTLD_NOW);
+#ifdef _AIX
+        snprintf(modulename, sizeof(modulename), "%s%s(%s%s.%d)",name,".a",name,LT_MODULE_EXT,LIBCLAMAV_MAJORVER);
+        rhandle = dlopen(modulename, RTLD_NOW | RTLD_MEMBER);
+#endif /* AIX */
```

Comment:
GitHub isn't allowing me to make a suggestion here because of the deleted line between `sprintf` and `rhandle`. 🤷  But you should do the same as I suggested above, with `#ifdef` + `#else`, and also cleaning up the formatting.